snuba/datasets/storage.py

```diff
@@ -21,6 +23,19 @@ class Storage(ABC):
     for more useful abstractions.
     """
 
+    def __init__(self, storage_key: StorageKey, storage_set_key: StorageSetKey):
+        self.__storage_key = storage_key
+        self.__storage_set_key = storage_set_key
+
+    def get_storage_key(self) -> StorageKey:
+        return self.__storage_key
+
+    def get_storage_set_key(self) -> StorageSetKey:
```

Comment:
nit: just get_storage_set. storage_key works for the storage, where the key is only a reference to the storage. the storage_set is entirely identified by the enum value instead, no need to call it a key.

Reply:
Well, it's more to match the name of the class which is still called `StorageSetKey` (despite the fact there is no `StorageSet` class)

Reply:
ok